recipe/doc/end_user_run_guide.md

```diff
@@ -0,0 +1,121 @@
+In particular, how these channels may or may not coexist will depend on the versions of CUDA that you need support for.
+
+### Pre-CUDA 12:
+
+Prior to CUDA 12, the only package available on conda-forge was the `cudatoolkit` package, a community-maintained, monolithic package containing the entire repackaged CTK.
+During the CUDA 11 release cycle, NVIDIA began maintaining a set of CUDA Toolkit packages in the nvidia channel.
+Unlike the monolithic conda-forge package, the nvidia channel distributed the CTK split into components such that each library was given its own package.
+This package organization made it possible to install separate components independently and better aligned the conda packaging ecosystem with other package managers, such as those for Linux distributions.
+However, this organization introduced a number of changes that were at times confusing -- such as the introduction of a `cuda-toolkit` (note the hyphen) metapackage that installs a partially overlapping set of components to the original `cudatoolkit` -- and at other times breaking, particularly in conda environments configured to pull packages from both conda-forge and the nvidia channel.
+Therefore, in a CUDA 11 world the conda-forge and nvidia channels were difficult to use in the same environment without some care.
+
+### CUDA 12.0-12.4
+
+With the CUDA 12 release, NVIDIA contributed the new packaging structure to conda-forge, introducing the same set of packages that existed on the nvidia channel as a replacement for the old `cudatoolkit` package on conda-forge.This was done starting with CUDA 12.0 to indicate the breaking nature of these changes compared to the prior CUDA 11.x packaging in conda-forge.
+These packages became the standard mechanism for delivering CUDA conda packages.
+Due to the scale of the reorganization, the CUDA 12.0, 12.1, and 12.2 releases also involved numerous additional fixes to the packaging structure to better integrate them in the Conda ecosystem.
+Due to the number of such changes that were required and the focus on improving the quality of these installations, during this time period no corresponding updates were provided for packages on the nvidia channel.
+While the conda-forge and nvidia channel package lists were the same (i.e. the same packages existed in both places with the same core contents like libraries and headers), the nvidia channel did not include many of the incremental fixes made on conda-forge to improve things like symlinks, static library handling, proper package constraints, etc.
+As a result, nvidia and conda-forge CUDA packages remained incompatible from CUDA 12.0-12.4.
+
+### CUDA 12.5+
+
+With CUDA 12.5, the nvidia channel was fully aligned with conda-forge.
+Packages on both channels are identical, ensuring safe coexistence of the two channels within the same conda environment.
```

Comment:
Let's qualify this. "Identical" implies they're binary-equivalent. To the best of my understanding, the `nvidia` packages are built from the same recipes and should be compatible but are not expected to be binary-equivalent.
```suggestion
Packages on both channels are nearly identical, ensuring safe coexistence of the two channels within the same conda environment.
```

Reply:
What would make them binary-equivalent according to your definition, @bdice?

Reply:
I think they would be binary-equivalent if not for the RPATH modifications that we do. The nvidia and conda-forge packages are just repackaging the same binaries built for the redist, so they should be otherwise binary-equivalent. I don't think the RPATH modification is relevant for users to know, so I think calling them identical is fine.

Reply:
We discussed this offline. We do expect equivalence here.